impl/sql/domainstorage/storage.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package domainstorage implements the admin.Storage interface.
+// Package domainstorage implements the domain.Storage interface.
```

Comment:
How about renaming `impl/sql/domainstorage` -> `impl/sql/domain` too?

Reply:
I thought about it.  Happy to do it. It's just that that would produce `domain.New` vs. `domainstorage.New`.  Should it be `domain.NewStorage`?